.eslintrc.json

```diff
@@ -26,6 +26,7 @@
         "no-constant-condition": ["error"],
         "no-else-return": ["error"],
         "object-curly-spacing": ["warn", "always"],
+        "prefer-const": ["error"],
```

Comment:
Do you think this should be error rather than just warning?

Reply:
A warning is better, I will update it

Reply:
👍🏻